docs/bestpractice.rst

```diff
@@ -4,4 +4,33 @@
 Best Practices for Collecting Physiological Data
 ================================================
 
-Coming soon.
\ No newline at end of file
+**Why collect physiological data?**
```

Comment:
It's not clear if you want just bold, or you want to create subtitles - in the second case, can you adapt them following [this document](https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst#section-structure)? It would make it more clear even in the source code!
A cool thing that you can also add is an anchor before every title, so that we can later link to each specific part! Look, for instance, [here](https://stackoverflow.com/a/19543591).

Reply:
I intentionally made the text bold. Would you like it to be italic headings with only the document title in bold? I can change it to that if you would like - let me know!